.github/workflows/test.yml

```
@@ -68,8 +59,40 @@ jobs:
       - name: Run tox
         run: |
             tox
-            coveralls
         env:
           TOXENV: ${{ matrix.toxenv }}
-          COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Store test coverage
+        uses: actions/upload-artifact@v2
+        with:
+          name: coverage
+          path: .coverage.*
+
+  coverage:
+    name: coverage
+    runs-on: ubuntu-latest
+    needs:
+      - test
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
```

```suggestion
          python-version: "3.8"
```

In tox.ini, the coverage environment has `basepython=python3.8`, so `tox -e coverage` doesn't actually do anything. See for example [this check](https://github.com/cfpb/wagtail-treemodeladmin/actions/runs/3794960410/jobs/6453593659) which, sadly, passes even though there's an error `coverage: skipped because could not find python interpreter with spec(s): python3.8`.

I'm surprised tox doesn't return a non-zero return code on that failure; per the documentation at https://tox.wiki/en/4.1.1/config.html#skip_missing_interpreters I would expect this to default to false, but maybe that's not the case. It seems like we have the same situation as https://github.com/beartype/beartype/commit/23ee35f5e80cdb016c78083abe6d243e914845e1 and maybe need to explicitly set that to false for this to fail.

Regardless, if we set the right Python version it should actually make the coverage tests run.